spec/support/sections/calendar.rb

```diff
@@ -4,6 +4,10 @@ class Calendar < SitePrism::Section
     elements :events, '.fc-event'
     elements :resource_cells, '.fc-resource-cell'
 
+    def appointments
+      background_events('appointment')
```

Comment:
Was thinking about how this is growing into something a little nasty. That JS stuff digs right down into the calendar internals rather than verifying on something closer to / the DOM elements. Just thinking out loud here and not necessarily a blocker but it would be nice to cut this down into something closer to the actual user experience, rather than the JS internals.